app/views/episode_transcripts/_form_status.html.erb

```diff
@@ -12,6 +12,6 @@
         <%= form.submit t(".update"), class: "btn btn-primary", data: data %>
       <% end %>
     </div>
-    <p class="status-text mt-2 mb-0 text-end"><strong><%= t(".updated_at_hint") %>:</strong> <%= l(episode.updated_at) %></p>
+    <p class="status-text mt-2 mb-0 text-end"><strong><%= t(".updated_at_hint") %>:</strong> <%= local_time_ago(form.object.updated_at) %></p>
```

Comment:
Wait, but what is `form.object`?  Up in `_form.html.erb`, it looks like it's configured to be the episode, not the transcript.

Bit tricky to confirm (since we have `touch: true` on all the child models, so timestamps trickle up).  But you could go save the episode main form... and make sure this tab shows an _older_ updated_at.